judgments/utils.py

```diff
@@ -190,14 +190,23 @@ def parse_parameter_as_int(params, parameter_name, default=None):
         return default
 
 
-def parse_date_parameter(params, param_name, default_to_last=False) -> Optional[date]:
+def parse_date_parameter(
```

Comment:
I think we're perhaps starting to hang too much on this function. I'm happy with this for now, but we should consider this as a candidate for refactoring how we validate the fields (eg by splitting "is this year valid" into its own validator)